# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/quota/moz.build
# Commit: 9987ebf11f14
# Full Hash: 9987ebf11f141d7e31c958972aee01676d717246
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2025-06-16 09:38:11
# Regressor Bug: 1962133
# File Overlap Count: 14
# Overlapping Files: dom/quota/CommonMetadata.h, dom/quota/DirectoryMetadata.h, dom/quota/test/gtest/TestQuotaManager.cpp, dom/quota/OriginInfo.cpp, dom/quota/QuotaManager.h
# Description:
#   Bug 1962133 - QM: Add accessed flag to FullOriginMetadata and persist it in the origin metadata file; r=dom-storage-reviewers,hsingh
#   
#   This patch introduces a new mAccessed field in OriginStateMetadata, which is
#   inherited by FullOriginMetadata. One of the reserved 32-bit fields in the
#   metadata header is now used as a bitfield, with one bit allocated to represent
# ==============================================================================

diff -r d2ec302b7103 -r 9987ebf11f14 dom/quota/moz.build
--- a/dom/quota/moz.build	Sun Jun 15 21:21:24 2025 +0000
+++ b/dom/quota/moz.build	Sun Jun 15 23:24:14 2025 +0000
@@ -57,6 +57,7 @@
     "DirectoryLockCategory.h",
     "DirectoryLockImpl.h",
     "DirectoryLockInlines.h",
+    "DirectoryMetadata.h",
     "DummyCipherStrategy.h",
     "EncryptedBlock.h",
     "EncryptingOutputStream.h",